src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -35,106 +35,110 @@ These are the basic steps to add UI Options to your website:
+2. Unzip the contents of the downloaded ZIP file to a location within your project. This will result in a new directory `infusion`. Note: In this guide we will use the directory `my-project/lib/`.
+3. Your `infusion` folder will include a single file containing all of the JavaScript you need (`infusion-uiOptions.js`), HTML templates, CSS files, and other components to get UI Options to work. You will later link to these files in your HTML files.
+4. Now that `infusion` is in your project directory, you can delete the `infusion-uiOptions-2.0.0.zip` (or similar name) from your download directory.
 
 ## Prepare your page ##
 
 ### The Sliding Panel ###
 
-The UI Options component includes HTML templates for all the controls, so you don't need to create any HTML for them. You only need to add a small amount of markup to the top of your page to tell UI Options where to render itself.
+The UI Options component includes HTML templates for all the controls, so you don't need to create any HTML for them. You only need to add a small amount of markup to the top of your webpage to tell UI Options where to render itself.
 
-Add the following markup as the first thing within your `<body>` tag:
+Add the following markup at the very beginning within your `<body>` tag to your page html:
 
 ```html
-<div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
-    <!-- This is the div that will contain the Preference Editor component -->
-    <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
-    <!-- This div is for the sliding panel that shows and hides the Preference Editor controls -->
-    <div class="fl-panelBar">
-        <span class="fl-prefsEditor-buttons">
-            <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
-            <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
-        </span>
+<body>
+    <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+        <!-- This is the div that will contain the Preference Editor component -->
+        <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+        <!-- This div is for the sliding panel that shows and hides the Preference Editor controls -->
+        <div class="fl-panelBar">
+            <span class="fl-prefsEditor-buttons">
+                <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+            </span>
+        </div>
     </div>
-</div>
+</body>
 ```
 
 The main `<div>` in this snippet contains two things:
 
-1. a `<div>` where an iframe will be inserted, containing the Fat Panel UI Options controls, and
+1. a `<div>` where an iframe will be inserted, containing the UI Options adjusters, and
```

Comment:
Maybe for all of these `<div>` elements include the class names so that they are easier to identify in the markup example?